cvat/apps/dataset_manager/formats/yolo.py

```diff
@@ -23,6 +23,9 @@ def _export(dst_file, temp_dir, instance_data, save_images=False):
     dataset = Dataset.from_extractors(GetCVATDataExtractor(
         instance_data, include_images=save_images), env=dm_env)
 
+    for item in dataset: # I don't konw why.
+        print(f"{item.subset}")
+
```

Comment:
I think we don't need it. Moreover, this issue exists for other formats as well.

Consider adding initialization of `self.subset` for `CvatTaskOrJobDataExtractor` from `cvat/apps/dataset_manager/bindings.py` instead.

Maybe something like this will work:
```python
        # bindings.py L1379
        dm.SourceExtractor.__init__(
            self,
            subset=instance_meta["subset"],
            media_type=dm.Image if dimension == DimensionType.DIM_2D else PointCloud
        )
```